akka-actor-typed-tests/src/test/scala/akka/actor/typed/TimerSpec.scala

```diff
@@ -141,13 +144,20 @@ class TimerSpec extends TestKit("TimerSpec")
       val ref = spawn(behv)
       probe.expectMessage(Tock(1))
       ref ! Cancel
+      probe.fishForMessage(3.seconds) {
```

Comment:
Shall we make it have a hint always? (string to explain what we wait for), I found that very helpful

Reply:
Hmm, you can put that in your failure message with FishingOutcomes.Fail, so I don't think it is needed anymore? (Additionally I decided to always include the seen messages up to the fail in the test failure, so maybe that also helps)

Reply:
I can't include any hints for the cases: timeout, match error, only for explicit fail

Reply:
Fair point, I'll add it